canard_internals.h

```diff
@@ -117,7 +117,7 @@ CANARD_INTERNAL int16_t descatterTransferPayload(const CanardRxTransfer* transfe
 
 CANARD_INTERNAL bool isBigEndian(void);
 
-CANARD_INTERNAL void swapByteOrder(void* data, unsigned size);
+CANARD_INTERNAL void swapByteOrder(void* data, size_t size);
```

Comment:
the argument that is passed in is a uint8_t, so we could change to that, but why would we make it a size_t?

Reply:
I agree that in this specific context a uint8_t could be used because the size is bounded to 8 bits. However, I'm more in favor of size_t since it naturally represents a size in general. It's self describing and would avoid any cast warning/errors fir the well known use case of sizeof(my_buffer).

Reply:
@tridge Any update on this? thanks!

Reply:
i think uint8_t is clearer, as that is the actual type used in all cases